### ISSUE TITLE
Malformed flow-version directory name

### ISSUE THREAD

**Issue by jamsea** [flowtype/flow-typed#610]
@orlandoc01 

I'm getting this error:
```
UNCAUGHT ERROR: Error: npm/bluebird_v3.x.x/flow_v0.32: Malformed flow-version directory name! Expected the name to be formatted as 'flow_all' or 'flow_-?v<VERSION>' or 'flow_v<VERSION>-?' or 'flow_v<VERSION>-v<VERSION>' (for a range).
```

When I run:
```
flow-typed install --overwrite
```

I'm guessing it's from your recent (about an hour-ish ago) commit in this folder:
https://github.com/flowtype/flow-typed/tree/master/definitions/npm/bluebird_v3.x.x

Let me know if you know what the exact issue is. If you're too busy I can open up a pull request later today.

Cheers

**Comment by orlandoc01** [comment]
Hmm, I ran the `validate-defs` command when testing which assured me the directories where properly formatted, but it appears the `flow-typed install` command parses the directory names differently. Anyway, it looks like someone has submitted a PR for it here #611 by renaming the folder.

**Comment by btipling** [comment]
I hope that PR is merged right away it breaks our CI pipeline. We cannot deploy and update flowtyped. :(

**Comment by shishkin** [comment]
Is there a workaround for this before #611 is merged?

**Comment by btipling** [comment]
Don't run flow-typed.

**Comment by shishkin** [comment]
I agree that update of `~/.flow-typed/repo/definitions` should be user's explicit decision.

**Comment by danharper** [comment]
This is fixed now (#611).
